src/lib/y2packager/widgets/product_license_translations.rb

```diff
@@ -126,6 +126,15 @@ def default_language
         return candidate_lang if translated
         DEFAULT_FALLBACK_LANGUAGE
       end
+
+      # Whether it is needed to force the default language
+      #
+      # @return [Boolean] true if running but not in a fbiterm; false otherwise
+      def force_language?
+        return false unless Yast::UI.TextMode
```

Comment:
You can simplify it to a single line and even avoid using the obsolete `Yast::Builtins` call:

```ruby
Yast::UI.TextMode && ENV["TERM"] != "iterm"
```